server/config.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+GITHUB_ACCESS_TOKEN = os.environ.get("GITHUB_ACCESS_TOKEN")
```

Comment:
Does this change how we run the tests or the server locally? Do we need a .env file?

Reply:
Updated it to so you can add a `.env` file in `server/` with the following line:

`GITHUB_ACCESS_TOKEN="YOUR_TOKEN"`